integration/docker/entrypoint.sh

```diff
@@ -47,12 +47,14 @@ alluxio_env_vars=(
 declare -A alluxio_env_violators=(
   ["ALLUXIO_UNDERFS_S3A_INHERIT_ACL"]="alluxio.underfs.s3a.inherit_acl"
   ["ALLUXIO_MASTER_FORMAT_FILE_PREFIX"]="alluxio.master.format.file_prefix"
+  ["AWS_ACCESSKEYID"]="aws.accessKeyId"
+  ["AWS_SECRETKEY"]="aws.secretKey"
 )
 
-for keyvaluepair in $(env | grep "ALLUXIO_"); do
+for keyvaluepair in $(env | grep -E "ALLUXIO_|FS_|AWS_"); do
```

Comment:
do you have a sense of how many alluxio properties do not start w/ the prefix "alluxio"?

Reply:
From the documentation there are 16 properties which are all UFS settings.

Reply:
i feel like all configuration properties should uniformly begin with the "alluxio" prefix. @apc999 do you think its feasible to deprecate and add the prefix "alluxio.underfs" to the existing properties which do not conform?.

Reply:
@apc999 ping :-)

Reply:
Talked with @apc999 @madanadit and I will update the shell script to validate the properties through a new Alluxio command.